src/components/pages/ProfilePage/ClientProfilePage.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { ProfileEditForm } from '../ProfileEdit';
+import profilepic from './profilepic.png';
+
+const ClientProfilePage = ({ user }) => {
+  const [profileEdit, setProfileEdit] = useState(false);
+
+  const handleChanges = e => {
+    setProfileEdit(true);
+  };
+  console.log(user);
```

Comment:
We should remove console logs from the components

Reply:
Done